docs/src/type_stability.md

```diff
@@ -215,5 +219,67 @@ Finally, let's look at the benchmarks
 @benchmark reshape_operator_data($((2, 2, 2)))
 ```
 
-Which is an innocuous but huge difference in terms of performance. Hence, we highly recommend using `Tuple` or `SVector` when defining the dimensions of a user-defined quantum object.
+Which is an innocuous but huge difference in terms of performance. Hence, we highly recommend using `Tuple` or `SVector` when defining the dimensions of a user-defined [`QuantumObject`](@ref).
+
+### The use of `Val` in some `QuantumToolbox.jl` functions
```

Comment:
maybe use subsub-section `##` instead of subsubsub-section `###` ?

Reply:
Yes, I did a typo